### PR TITLE
Ruru/resunga

### PR DESCRIPTION
**Pull request by shree404**

### CODE REVIEW

File: data/websites.json [hunk 1]
```diff
@@ -375,6 +375,26 @@
       "url": "https://rainadevichhaharamun.gov.np/",
       "description": "Official Website Of Rainadevi Chhahara Rural Municipality"
     },
+    {
+      "name": "Rainadevi Chhahara Rural Municipality",
```
Comment (TheLearneer) [inline data/websites.json:379]: This site already exists!

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Please remove the duplicate

**Review by TheLearneer — APPROVED** [review]
LGTM

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts again.
Thank you

**Pull request closed without merging**